sdlib/d/gc/extent.d

```diff
@@ -253,6 +256,32 @@ public:
 		return _metadata.slabData;
 	}
 
+	/**
+	 * Label features for slabs.
```

Comment:
labels?

Reply:
I have no idea what labels are supposed to be at this time.

Reply:
You also dump label in there withotu explaination. Nevertheless, the description says "has free space", and I don't see what's wrong with `hasFreeSpace(uint i)` has a method.